apps/payment/views.py

```diff
@@ -132,16 +123,11 @@ class PaymentTransactionViewSet(viewsets.ModelViewSet):
     """
 
     permission_classes = (permissions.IsAuthenticated,)
-
-    def get_serializer_class(self):
-        if self.action in ["list", "retrieve"]:
-            return PaymentTransactionReadOnlySerializer
-        if self.action == "create":
-            return PaymentTransactionCreateSerializer
-        if self.action in ["update", "partial_update"]:
-            return PaymentTransactionUpdateSerializer
-
-        super().get_serializer_class()
```

Comment:
Changing the way the viewsets get serializers only affects Swagger, or is the change based on something else?

Reply:
The functionality is not changed, it was just refactored to a helper method for defining multiple serializers on a viewset